Helm/opensearch/values.yaml

```diff
@@ -245,14 +262,14 @@ podSecurityContext:
 securityContext:
   capabilities:
     drop:
-    - ALL
+      - ALL
   # readOnlyRootFilesystem: true
   runAsNonRoot: true
   runAsUser: 1000
 
 securityConfig:
   enabled: true
-  path: "/usr/share/elasticsearch/plugins/opendistro_security/securityconfig"
+  path: "/usr/share/opensearch/plugins/opensearch-security/securityconfig"
```

Comment:
@DandyDeveloper Looks like we missed it while adding the security configurations. Good catch @hgoscenski-imanage .

Reply:
Ref : https://opensearch.org/docs/opensearch/install/docker-security/

Reply:
Yep, good find.